{"uuid": "4d810832-0854-4503-9fd4-bac12f165258", "children": ["7c8bac8b-97ce-407e-a460-44baa28a0f85", "da3361ab-16d7-4655-a0c4-0289c8b7fcc8", "428702d0-f41d-4e3d-8e22-30065cb9394a"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 26, in driver\n    driver = webdriver.Chrome(executable_path=(f\"{os.getcwd()}/chromedriver\"), options=options)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1718832815755, "stop": 1718832815755}], "start": 1718832815755, "stop": 1718832815771}